editor/progress_dialog.cpp

```diff
@@ -155,9 +155,10 @@ void ProgressDialog::_popup() {
 
 	Ref<StyleBox> style = get_stylebox("panel", "PopupMenu");
 	ms += style->get_minimum_size();
-	for (int i = 0; i < 4; i++) {
-		main->set_margin(Margin(i), style->get_margin(Margin(i)));
-	}
+	main->set_margin(MARGIN_LEFT, style->get_margin(MARGIN_LEFT));
+	main->set_margin(MARGIN_RIGHT, -style->get_margin(MARGIN_RIGHT));
```

Comment:
Won't there be many other occurrences in the code where the margin is expected to be an absolute value?

Isn't it a bit unintuitive to expect users to define a negative right and bottom margin?